README.md

```diff
@@ -19,3 +19,21 @@ Documents related to Code for Poznan organization.
 ## License
 
 Documents are licensed under [CC-BY-SA-4.0-PL](https://creativecommons.org/licenses/by-sa/4.0/deed.pl) (with the exception of Code for Poznan logo).
+
+## Welcome message - mail and messenger
+
+Cześć *Tu wpisz imię* !
+
+Kwarantanna nadal trwa i nie wiadomo kiedy wszystko wróci do normy, dlatego kontynuujemy nasze spotkania w wersji online.
+Jeśli chciałbyś/łabyś do nas dołączyć, zapraszamy na spotkanie organizacyjne, wprowadzające dla nowych uczestników, które odbędzie się w środę *tu wpisz dokładną datę* o 18:00 tutaj: [https://meet.google.com/pjv-yerm-cxn](https://meet.google.com/pjv-yerm-cxn)
+
+Zanim klikniesz w środę w link:
+1. Wypełnij proszę ten formularz: [https://forms.gle/VhCXKiwe3xv9y3Ss8](https://forms.gle/VhCXKiwe3xv9y3Ss8) - dopiszemy Twoje dane do naszej bazy uczestników.
+2. Dołącz do nas na Slacku, tu link z zaproszeniem: [https://join.slack.com/t//t/codeforpoznan/shared_invite/zt-8a7u52j8-yqB01C2YgYF4Lvd1pFM_jw](https://join.slack.com/t//t/codeforpoznan/shared_invite/zt-8a7u52j8-yqB01C2YgYF4Lvd1pFM_jw)
+3. Daj znać jakie masz doświadczenie w progamowaniu (jaki język)/projektowaniu/testowaniu, w czym się dobrze czujesz, albo czego się uczysz - będzie nam łatwiej znaleźć zadania w odpowiednim dla Ciebie zespole :)
+
+PS. Pracujemy na dockerach, a na Windowsie jest to niemożliwe. Najlepiej więc gdybyś miał/a już gotowe Ubuntu na swoim komputerze lub korzystał z MacOS. Normalnie pomoglibyśmy Tobie z wszelkimi konfiguracjami, ale w obecnych warunkach będzie to niestety niemożliwe. 
```

Comment:
- Bez kropki po PS. 
- Pracujemy na Dockerze / z Dockerem. 
- Na Windowsie jest to możliwe, ale nieco bardziej problematyczne. Windows 10 Pro wystartuje chyba bez problemów, na Homie można już też to obejść: https://docs.microsoft.com/en-us/windows/wsl/install-win10. 
- korzystał z MacOS - brak żeńskiej wersji (jak jest wyżej w przypadku posiadania Ubuntu)